AUTHORS.md

```diff
@@ -0,0 +1,4 @@
+Authors
+=======
+- [Antranig Basman](https://github.com/amb26)
+- [Leopold Burdyl](https://github.com/vilnius-leopold)
```

Comment:
As of https://github.com/GPII/linux/tree/76def8c4ef6fe5b5fa671f6285536f440770a428/node_modules/gsettingsBridge, they're still missing a few other contributors:
* Yura Zenevich
* Kasper Markus
* Colin Clark
* Javi Hernández